src/seedu/addressbook/commands/InfoCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.commands;
+
+public class InfoCommand extends Command{
+    public static final String COMMAND_WORD= "info";
+    public static final String MESSAGE_USAGE= COMMAND_WORD+": print out several information"+"\n"+"Example: "+COMMAND_WORD ;
+    public static final String INFO ="This is the addressbook level2 version 0.1 "
+    + "\n"+ "Developed by Zeng Hao(Oscar)";
+
+    public InfoCommand () {}
```

Comment:
You don't need to explicitly define the no-argument constructor though it is not wrong. Java itself will do it for you while compiling in case if you haven't defined any constructors.